src/BUILD

```diff
@@ -570,7 +570,7 @@ JAVA_TOOLS_DEPLOY_JARS = [
     "//conditions:default": [],
 })
 
-JAVA_VERSIONS = ("11",)
+JAVA_VERSIONS = ("11", "15")
```

Comment:
Remove "15" from here. This is for testing java_tools releases.

Reply:
Sorry, done

Reply:
Without this change, I cannot build java_toolchain 15.

I had to apply this patch:

```diff
diff --git a/src/BUILD b/src/BUILD
index 215b656fc8..419a122013 100644
--- a/src/BUILD
+++ b/src/BUILD
@@ -570,7 +570,7 @@ JAVA_TOOLS_DEPLOY_JARS = [
     "//conditions:default": [],
 })
 
-JAVA_VERSIONS = ("11",)
+JAVA_VERSIONS = ("11","15")
 
 [
     genrule(
```

To create `java_toolchain_java15.zip`:

```
  $ bazel build src:java_tools_java15.zip
INFO: Analyzed target //src:java_tools_java15.zip (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
Target //src:java_tools_java15.zip up-to-date:
  bazel-bin/src/java_tools_java15.zip
INFO: Elapsed time: 0.119s, Critical Path: 0.00s
INFO: 0 processes.
INFO: Build completed successfully, 1 total action
```

With that java_toolchain_java15.zip I can build and run with Java 15:

```shell
  $ bazel run java:Javac15Example
INFO: Analyzed target //java:Javac15Example (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
Target //java:Javac15Example up-to-date:
  bazel-bin/java/Javac15Example.jar
  bazel-bin/java/Javac15Example
INFO: Elapsed time: 0.090s, Critical Path: 0.01s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
INFO: Build completed successfully, 1 total action
Hello,
JDK 15
```

The source code for that test project is [here](https://github.com/davido/bazel_java_toolchain14).

Reply:
Sorry for the confusion.
You should create java_toolchain11.zip and with that you should be able to build either Java 11, Java 14 or Java 15.